src/state/routing/useRoutingAPITrade.ts

```diff
@@ -81,6 +88,8 @@ export function useRoutingAPITrade<TTradeType extends TradeType>(
     amount: amountSpecified,
     tradeType,
     routerPreference,
+    inputTax,
+    outputTax,
```

Comment:
you do a lot of work to pass the input and output tax to `src/state/routing/useRoutingAPITrade.ts` but could you just call `useSwapTaxes` in `useRoutingApiTrade` on `currencyIn` and `currencyOut`?

Reply:
once the BE handle this for us, we should be able to take all of the work bubbling up the taxes